Ticketing/Ticketing.UnitTests/ControllersTests/CartsControllerTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using AutoFixture;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Ticketing.BAL.Contracts;
+using Ticketing.BAL.Model;
+using Ticketing.DAL.Domain;
+using Ticketing.UI.Controllers;
+
+namespace Ticketing.UnitTests.ControllersTests
+{
+    public class CartsControllerTests
+    {
+        private readonly IFixture _fixture;
+
+        public CartsControllerTests()
+        {
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task GetAsync_Succes()
+        {
+            var collection = _fixture.Build<ShoppingCartReturnModel>()
+             .CreateMany(1)
+             .ToList();
+
+            Mock<ICartService> service = new Mock<ICartService>();
+
+            service.Setup(s => s.CartItemsAsync(It.IsAny<Guid>())).ReturnsAsync(collection);
+
+            var controller = new CartsController(service.Object);
+            var result = await controller.GetAsync(Guid.NewGuid());
+
+            service.Verify(u => u.CartItemsAsync(It.IsAny<Guid>()), Times.Once, "fail");
+            Assert.IsType<OkObjectResult>(result);
+        }
+
+        [Fact]
+        public async Task GetAsync_fail()
+        {
+            var collection = _fixture.Build<ShoppingCartReturnModel>()
+             .CreateMany(0)
+             .ToList();
+
+            Mock<ICartService> service = new Mock<ICartService>();
+
+            service.Setup(s => s.CartItemsAsync(It.IsAny<Guid>())).ReturnsAsync(collection);
```

Comment:
Please use real value instead of It.IsAny to prevent mutations.
In case of objects comparing use https://github.com/GregFinzer/Compare-Net-Objects package